airbyte-integrations/connectors/source-zendesk-support/source_zendesk_support/run.py

```diff
@@ -1,14 +1,48 @@
 #
-# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+# Copyright (c) 2024 Airbyte, Inc., all rights reserved.
 #
 
-
 import sys
+import traceback
+from datetime import datetime
+from typing import List
 
-from airbyte_cdk.entrypoint import launch
+from airbyte_cdk.entrypoint import AirbyteEntrypoint, launch, logger
+from airbyte_cdk.exception_handler import init_uncaught_exception_handler
+from airbyte_cdk.models import AirbyteErrorTraceMessage, AirbyteMessage, AirbyteTraceMessage, TraceType, Type
 from source_zendesk_support import SourceZendeskSupport
 
 
+def _get_source(args: List[str]):
```

Comment:
It feels like this method should be imported and reused from CDK, rather than be applied to each source / destination separately.

Do we consider this in the future?

Reply:
yes i think it would make sense given the repetition thusfar